pkg/server/endpoints/run.go

```diff
@@ -4,13 +4,17 @@ import (
 	"context"
 	"fmt"
 	"net"
-	"net/http"
 
 	"github.com/HewlettPackard/galadriel/pkg/common/util"
 	"github.com/HewlettPackard/galadriel/pkg/server/datastore"
 	"github.com/labstack/echo/v4"
 	"github.com/labstack/echo/v4/middleware"
 	"github.com/sirupsen/logrus"
+
+	"github.com/HewlettPackard/galadriel/pkg/common/telemetry"
+
+	adminAPI "github.com/HewlettPackard/galadriel/pkg/server/api/admin"
```

Comment:
```suggestion
	adminapi "github.com/HewlettPackard/galadriel/pkg/server/api/admin"
```

When renaming an imported package, the local name should follow the same guidelines as package names (lower case, no under_scores or mixedCaps).